### ISSUE TITLE
[SCREENREADER]: VAOS Scheduler - Certain alert messages should be announced to assistive technology

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#2863]
## Description

I found two places the alert messages are being added to screens dynamically. These are potential stops in the schedule flow, and should be announced to assistive technology users. Screenshots and code recommendations attached below.

## Point of Contact

**VFS Point of Contact:** _Trevor_

## Acceptance Criteria

* As a screen reader user, I want the alert boxes to be announced when I try to select a VA provider that I cannot schedule an appoint with, AND
* I want to hear the alert box announced when I load the detail view for a pending appointment.
* I should hear these messages announced in all major screen readers (JAWS, NVDA, VoiceOver) in their preferred browsers.

## Environment

* /vaos/new-appointment/va-facility
* ~~/vaos/appointments/pending/<UNIQUE_ID>~~

## Possible Fixes (optional)

We could add an empty `<div>` to the pages listed in the Environment above, and add `aria-live` and `aria-atomic` attributes so the alerts will be read aloud. I created a public codepen to show the different behaviors for live regions: https://codepen.io/tpierce_402/pen/JjjXJXp

```html
<div aria-atomic="true" aria-live="assertive">
  <!-- React <Alert /> component
</div>
```

## WCAG or Vendor Guidance (optional)

* [Error Identification: Understanding SC 3.3.1](https://www.w3.org/TR/UNDERSTANDING-WCAG20/minimize-error-identified.html)
* [Info and Relationships: Understanding SC 1.3.1](https://www.w3.org/TR/UNDERSTANDING-WCAG20/content-structure-separation-programmatic.html)

## Screenshots or Trace Logs

![Screen Shot 2019-10-29 at 12 13 30 PM](https://user-images.githubusercontent.com/934879/67792365-03823e00-fa47-11e9-9b04-a905ae561ae4.png)

---
![Screen Shot 2019-10-29 at 12 15 35 PM](https://user-images.githubusercontent.com/934879/67792379-08df8880-fa47-11e9-9611-dae47ff3ffb7.png)

**Comment by jenniemc** [comment]
based on design changes, we don't have Pending pages anymore
